lib/eunit/test/tutf8.erl

```diff
@@ -0,0 +1,12 @@
+%% coding: utf-8
+
+-module(tutf8).
+
+-include_lib("eunit/include/eunit.hrl").
+
+foo_test_() ->
+	[
+ 	  {<<"_göod">>, fun() -> io:format("goöd"), io:format([128,64]), ?assert("gö"=="gö") end}
```

Comment:
I'm guessing it's just that same distinction as with Erlang strings. `<<"_göod">>` is equivalent to `<<$_, $g, $ö, $o, $d>>`, which assumes every single term is an integer (codepoint). This in turns means they're mod 255. The `/utf8` filter likely asks for a proper handling and conversion to utf8 bytes whereas default just yields codepoints.